configuration/config2/SchemaValue.cpp

```diff
@@ -282,6 +281,38 @@ void SchemaValue::getAllowedValues(std::vector<AllowedValue> &allowedValues, con
         {
             allowedValues.push_back({ *it, "" });
         }
+
+        if (!allowedValues.empty() && m_valueLimitRuleType == "addDependencies_1")
+        {
+            std::vector<std::string> params = splitString(m_valueLimitRuleData, ",");
+            std::string matchPath = params[0];
+            std::string matchAttribute = params[1];
```

Comment:
is it safe to reference params[1-3]? are we guaranteed params.length > 1?

Reply:
Exceptions added.